### ISSUE TITLE
JavaFX virtual keyboard won't open on CodeArea

### ISSUE THREAD

**Issue by OmriSoudry** [TomasMikula/RichTextFX#332]
I'm using a touchscreen device with no keyboard, and the JavaFX virtual keyboard won't show up when I focus on the CodeArea.

**Comment by JordanMartinez** [comment]
That feature isn't supported due to simplicity. There's a brief note about this [in the wiki](https://github.com/TomasMikula/RichTextFX/wiki/Dropped-Features)

**Comment by TomasMikula** [comment]
@OmriSoudry it was just too much to support all those TextArea features from the start, if I wanted to get something working in reasonable time. Do you want to explore how to add support for virtual keyboard?

**Comment by JordanMartinez** [comment]
Most of those "dropped" features have been requested. I think this issue actually finishes the list.

**Comment by OmriSoudry** [comment]
@TomasMikula Yes, I'd like to explore ways to do that, because I had to implement an external keyboard, and it's not working as it should.

**Comment by TomasMikula** [comment]
Sounds great!
